pkg/vm/vm.go

```diff
@@ -1459,7 +1459,6 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 		case *stackitem.Struct:
 			item.Remove(index)
 		}
-		v.refs.Remove(elem)
```

Comment:
It even had a separate designated review comment (https://github.com/nspcc-dev/neo-go/pull/1670/commits/324107b31ee113a60d7739fee867e01beb30c298#r559602943), but I'm also not sure why this line was added.